pkg/issuer/acme/dns/clouddns/clouddns_test.go

```diff
@@ -110,7 +110,11 @@ func TestLiveGoogleCloudCleanUp(t *testing.T) {
 
 func TestDNSProvider_getHostedZone(t *testing.T) {
 	testProvider, err := NewDNSProviderCredentials("my-project", util.RecursiveNameservers, "test-zone")
-	assert.NoError(t, err)
+	if err != nil {
```

Comment:
Instead of skipping on error which might hide real errors, can we not re-use the `gcloudLiveTest` check which is used elsewhere?

https://github.com/jetstack/cert-manager/blob/c2634d3538bdaccd602c59f1d25335c871d05e48/pkg/issuer/acme/dns/clouddns/clouddns_test.go#L83

Reply:
that totally makes sense, yeah; I (maybe wrongly) assumed that `gcloudLiveTest` wasn't used in `TestDNSProvider_getHostedZone` for a reason.

If you think that's the way to go, I'll make that change.

Reply:
(I think my assumption was based on the fact that the tests above the one I changed have "Live" in the title)

Reply:
Yeah, I think it was omitted by mistake (?). A quick look through blame, this test func was added at a much later date (~3 years).

Reply:
As long as it still does the right thing for you, let's go with the gcloud live test check.